taxonomies/categories.txt

```diff
@@ -1708,21 +1780,22 @@ en:distilled anis
 
 #<en:Cypriot distilled anis
 #<en:Greek distilled anis
-el, el:Ouzo, Oύζο
-protected_name_file_number:en: PGI-CY+GR-01828
-protected_name_type:en: gi
-origins:en: en:Cyprus, Greece
 
 <en:Hard liquors
 en:Tequilas
 de:Tequilas
 es:Tequilas
+fi:Tequilat
 fr:Tequilas
 hu:Tekilák
 it:Tequila
 nl:Tequilas, Tequila
 nl_be:Tequilas
 ro:Tequilla
+el, el:Ouzo, Oύζο
+protected_name_file_number:en: PGI-CY+GR-01828
+protected_name_type:en: gi
+origins:en: en:Cyprus, Greece
```

Comment:
Doesn't look like it belongs under Tequila.

Reply:
fixed